Update card

jasonkim-jk opened the pull request:
![captured (18)](https://user-images.githubusercontent.com/63427860/87867859-fa732d00-c945-11ea-938c-45d0c8784a20.gif)

jasonkim-jk committed:
added edit button on each card

jasonkim-jk committed:
updated a basic UpdateCard component

jasonkim-jk committed:
updated code
- added updateView prop in ViewCards state
- added handleUpdate, updateCardData method to interect with UpdateCard component

jasonkim-jk committed:
completed update card
- modified update related methods

thebearingedge commented on dist/style.css:
```diff
@@ -5,12 +5,14 @@
 
 .view-card-container,
 .review-card-container,
+.update-card-container,
```
Looks like a lot of CSS classes need these styles. Can you use the bootstrap container instead?

thebearingedge commented on src/components/app.jsx:
```diff
@@ -63,7 +70,7 @@ export default class App extends React.Component {
 
   render() {
-    console.log("Cards From App: ", this.state.cards)
+    // console.log("Cards From App: ", this.state.cards)
```
This can be deleted

jasonkim-jk committed:
modified from css style to bootstrap style
- moved from class selector's styles to each react component's bootstrap styles or inline styles
